composer.json

```diff
@@ -13,19 +13,17 @@
         "bin-dir": "bin"
     },
     "require": {
-        "php": "^7.4",
+        "php": ">=8.0",
         "predis/predis": "^1.1.1",
         "doctrine/cache": "~1.3",
-        "b1rdex/predis-compressible": "^0.5.0"
+        "b1rdex/predis-compressible": "^0.6.0"
     },
     "require-dev": {
-        "m6web/php-cs-fixer-config": "^1.0",
-        "atoum/atoum": "^3.3.0",
-        "atoum/atoum-bundle": "^2.0",
+        "m6web/php-cs-fixer-config": "^2.0",
+        "atoum/atoum": "^4.0.0",
         "atoum/stubs": "~2.5",
-
         "m6web/redis-mock": "@stable",
-        "symfony/symfony": "~5.0",
+        "symfony/symfony": "~5.0 || ~6.0",
```

Comment:
This bundle uses a bunch of symfony component, why isn't it in `require` ? (see #63, maybe there's a good reason, I just don't know :grin:  )